File: Lecture008 Switch Statement & Functions/calculator.java
import java.util.*;

class calculator {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        while(1 == 1) {

        System.out.println();
        System.out.print("Enter first number: ");
        int a = sc.nextInt();
    
        System.out.print("Enter second number: ");
        int b = sc.nextInt();

        System.out.println("Please select an operation from the menu below: ");
        System.out.println("1. Addition");
        System.out.println("2. Subtraction");
        System.out.println("3. Multiplication");
        System.out.println("4. Division");
        System.out.println("5. Modulus");
        System.out.println("6. Exit");

        System.out.print("Enter your choice: ");
        int choice = sc.nextInt();

        switch(choice) {
            case 1: System.out.println("Sum of " + a + " and " + b + " is: " + (a+b));
            break;

            case 2: System.out.println("Difference of " + a + " and " + b + " is: " + (a-b));
            break;

            case 3: System.out.println("Product of " + a + " and " + b + " is: " + (a*b));
            break;

            case 4: System.out.println("Division of " + a + " and " + b + " is: " + (a/b));
            break;

            case 5: System.out.println("Modulus of " + a + " and " + b + " is: " + (a%b));
            break;

            case 6: System.exit(-1);
            break;

            default: System.out.println("Invalid Input!");
        }

        }
    }    
}

File: Lecture008 Switch Statement & Functions/calculator.cpp
#include<iostream>
using namespace std;
int main()
{
    int ch;
    cout<<"Enter 1 for addition 2 for substraction 3 for multipliaction 4 for divison : ";
    cin>>ch;
    int x,y;
    cout <<"Enter the values of x : ";
    cin>>x;
    cout <<"Enter the values of y : ";
    cin>>y;
    int a=x+y;
    int s=x-y;
    int m=x*y;
    int d=x/y;
    int r=x%y;
    switch(ch)
    {
        case 1: cout<<"Addition : "<<a;
                break;
        case 2: cout<<"Substraction : "<<s;
                break; 
        case 3: cout<<"Multipication : "<<m;
                break;
        case 4: cout<< "Divison : "<<d;
                break;
        case 5: cout<<"Modulo : "<<r;
                break; 
       default: cout<<"invalid input";
                break;                  
    }
    return 0;
}
